sorc/grid_tools.fd/regional_esg_grid.fd/psym2.f90

```diff
@@ -299,19 +286,18 @@ subroutine expsym2d(x,z,zd)!                                         [expsym2]
 z=z*exp(trxh)
 end subroutine expsym2d
 
-!=============================================================================
+!>
```

Comment:
Can you write a one-sentence description of this subroutine?

Reply:
There are grid_tools.fd included 4 directories and 12 f90 source codes. Still working on it